packages/shared/routes/setup/ledger/views/AccountIndex.svelte

```diff
@@ -9,11 +9,20 @@
     export let locale
     export let mobile
 
+    let expert = false
     let loading = false
 
+    let min = 0
+    let max = 2147483647
```

Comment:
@maxwellmattryan Why this number?

Reply:
I believe this is what the max index was in Trinity.